installers/omnibus/components/sdlc/config/config.memory.slim.yml

```diff
@@ -0,0 +1,76 @@
+server:
+  adminContextPath: /admin
+  applicationConnectors:
+    - type: http
+      port: ${LEGEND_OMNIBUS_SDLC_PORT}
+      maxRequestHeaderSize: 128KiB
+  adminConnectors:
+    - type: http
+      port: ${LEGEND_OMNIBUS_SDLC_ADMIN_PORT}
+  gzip:
+    includedMethods:
+      - GET
+      - POST
+  requestLog:
+    type: classic
+    level: "OFF"
+    appenders:
+      - type: console
+        logFormat: "OFF"
+  rootPath: /api
+sessionCookie: LEGEND_SDLC_JSESSIONID
+logging:
+  level: INFO
+  appenders:
+    - type: console
+      logFormat: "%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p [%thread] %c - %m%n"
+swagger:
+  resourcePackage: org.finos.legend.sdlc.server.resources
+  title: Legend SDLC (In Memory)
+  version: local-snapshot
+  schemes: []
+
+pac4j:
+  clients:
+    - org.pac4j.core.client.direct.AnonymousClient: {}          
+  bypassPaths:
+    - /api/info
+    - /api/server/info
+    - /api/server/platforms
+    - /api/auth/authorized      
+
+applicationName: Legend SDLC
+filterPriorities:
+  GitLab: 1
+  org.pac4j.j2e.filter.CallbackFilter: 2
+  org.pac4j.j2e.filter.SecurityFilter: 3
+  CORS: 4
+cors:
```

Comment:
don't think we need this `cors` block.
Also, could you do me a favor and make sure the order of things in this file is very similar to that of other SDLC config files - I figured people might take a look at this for comparison between different modes and minimizing the diff would be great for them